viewer/src/workers/parser.worker.ts

```diff
@@ -108,10 +108,15 @@ export class ParserWorker {
     const rust = await rustModule;
     // TODO handle parsing failure
     const ctm = rust.parse_ctm(buffer);
+    const indices = ctm.indices();
+    const vertices = ctm.vertices();
+    const normals = ctm.normals();
     const result = {
-      indices: ctm.indices(),
-      vertices: ctm.vertices(),
-      normals: ctm.normals()
+      indices,
+      vertices,
+      normals,
+      colors: new Float32Array(indices.length),
```

Comment:
Doesn't this buffer need to be 3*indices.length? Also, note that we should look into (separate from this PR) is to replace with Uin8

Reply:
@dragly